### DISCUSSION TITLE
Android Studio wants to upgrade APG ... but cannot.

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#281] [Q&A]
Android Studio wanted to upgrade to Ladybug... so I did.  Android Studio Ladybug | 2024.2.1 Patch 2

Maybe you can help me understand this Android Studio message - it thinks I should upgrade to APG 8.7.2  But it cannot seem to find the file to upgrade.  I don't know much about this...  help?

<img width="1038" alt="Screenshot 2024-11-03 at 6 52 57 PM" src="https://github.com/user-attachments/assets/86f50df6-3b9b-49ca-b4c3-f45420bd40d3">

**Comment by marcprux** [comment]
I haven't seen that before. Is it blocking you from running in Studio altogether, or can you dismiss the Upgrade checkbox?

We will eventually upgrade the AGP dependency, but we usually like some time to test before jumping on the latest release, and Studio/Gradle have generally had good backwards-compatibility support for using older versions, so I'd be surprised if there isn't some way to disable this check.

**Comment by davidakoontz** [comment]
Ok - no it is NOT blocking. I can ignore it. It seems I’ve got many versions and don’t know how to manage them all. David 

- On Nov 4, 2024, at 3:08 PM, marcprux wrote:﻿

I haven't seen that before. Is it blocking you from running in Studio altogether, or can you dismiss the Upgrade checkbox?
We will eventually upgrade the AGP dependency, but we usually like some time to test before jumping on the latest release, and Studio/Gradle have generally had good backwards-compatibility support for using older versions, so I'd be surprised if there isn't some way to disable this check.
